maintainers/maintainer-list.nix

```diff
@@ -3770,12 +3770,6 @@
     githubId = 57923898;
     name = "Elyhaka";
   };
-  em0lar = {
```

Comment:
What the hell is happening here? 70 files??

Reply:
I did `git rebase --interactive 9a995f5c6a0b7e782017b725eb8a34ff0d7de527` squashing all but the last commit. Not sure what went wrong... :disappointed: 

![image](https://user-images.githubusercontent.com/106774777/174091546-a63380c5-4a46-44f7-86d1-7186a67e9184.png)